src/utils/hooks/use-get-accounts.tsx

```diff
@@ -0,0 +1,33 @@
+
```

Comment:
Note: this is a quick fix to prevent having to duplicate the logic for fetching accounts. I've left the function exactly as it is to minimise the risk of regression, and moved it to a hook so that we only need to write the function once.

We're still running the function twice (unnecessarily) but this will be fixed when we've had the discussion about state handling which is scheduled for next sprint.